src/service/xhr-impl.js

```diff
@@ -109,10 +109,7 @@ export class Xhr {
     /** @const {!Window} */
     this.win = win;
 
-    const ampdocService = Services.ampdocServiceFor(win);
-    /** @private {?./ampdoc-impl.AmpDoc} */
-    this.ampdocSingle_ =
-        ampdocService.isSingleDoc() ? ampdocService.getAmpDoc() : null;
+    this.ampdocService = Services.ampdocServiceFor(win);
```

Comment:
The `isSingleDoc` is still necessary ([context](https://github.com/ampproject/amphtml/pull/9350#discussion_r116625881)).

Also, leave this as a private field, they get minified better.

Reply:
hey justin, i know we discussed this a bit yesterday but it might have been under a different context. Do we really need this private field. in my changes you can see that i check if amp is single doc in the fetch call and pass along that info to the methods that need that info. I dont think this private field is needed at all.